tests/test_utils.py

```diff
@@ -275,6 +275,17 @@ def test_latest_github_release(self, mock_get):
             with open('./cache.txt', 'r') as f:
                 self.assertEqual(f.read(), 'abc')
 
+    def test_read_release_data(self):
+        with open('./cache.txt', 'w') as f:
```

Comment:
This file will not be cleaned from the filesystem after the test. You can use `with self.runner.isolated_filesystem()`